src/workflow-design-guide/general-principles.rst

```diff
@@ -653,22 +653,14 @@ Automating Failure Recovery
 Job Submission Retries
 ^^^^^^^^^^^^^^^^^^^^^^
 
-When submitting jobs to a remote host, use job submission retries to
-automatically resubmit tasks in the event of network outages. Note this is
-distinct from job retries for job execution failure (just below).
+When submitting jobs to a remote host, use a platform with
+job submission retries to automatically resubmit tasks in the event
```

Comment:
Job submission retries (like execution retries) can not be configured on platforms. This is still `[runtime]` functionality.